src/extras/ContextMenu/ContextMenu.js

```diff
@@ -869,6 +870,22 @@ class ContextMenu {
         }
     }
 
+    _updateSubMenuInfo() {
+        if (!this._context) return;
+        let showOnLeft = null;
+        this._itemList.forEach((item) => {
+            if (item.subMenu) {
+                const itemElement = item.itemElement;
+                if (!showOnLeft) {
+                    const itemRect = itemElement.getBoundingClientRect();
+                    const subMenuWidth = 200; // TODO
```

Comment:
We could set subMenuWidth in ContextMenu cfg ? 
Something like 
`this.subMenuWidth = cfg.subMenuWidth || 200;`
in constructor

Reply:
I have updated it so that it will automatically read the width of submenu and adjust the position accordingly. No need to provide the width in constructor.